src/css/custom.css

```diff
@@ -76,3 +76,111 @@ img + br + em::before {
     flex: 1;
     min-width: 20rem;
 }
+
+.typography h1 {
+    font-family: "Segoe UI", "Helvetica Neue", "Droid Sans", Helvetica, Arial,
```

Comment:
Can you group all of the reused style declarations to reduce the repetition of things like the font family, font weights, padding, etc.?

Reply:
CSS is temporary.

Reply:
Updated.